MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController+Design.swift

```diff
@@ -0,0 +1,114 @@
+import UIKit
+
+extension MovieDetailViewController: DesignProtocol {
+    
+    func buildViews() {
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+    
+    func createViews() {
+        scrollView = UIScrollView()
+        view.addSubview(scrollView)
+        
+        contentView = UIView()
+        scrollView.addSubview(contentView)
+        
+        let details: (() -> DetailTitleViewModel?) = { [weak self] in
+            guard let self = self else { return nil }
+            
+            return self.presenter.getMovieDetails(for: self.movieId)
+        }
+        titleView = DetailTitleView(getDetailsFunction: details)
+        contentView.addSubview(titleView)
+        
+        overviewTitle = UILabel()
+        contentView.addSubview(overviewTitle)
+        
+        overview = UILabel()
+        contentView.addSubview(overview)
+        
+        let cast: (() -> [CastViewModel]) = { [weak self] in
+            guard let self = self else { return [] }
+            
+            return self.presenter.getCast(for: self.movieId)
+        }
+        castView = CastView(getCast: cast)
```

Comment:
Swift allows you to write trailing closures without defining the parameter.
```swift
        castView = CastView { [weak self] in
            guard let self = self else { return [] }
            
            return self.presenter.getCast(for: self.movieId)
        }
```

Reply:
[Here](https://github.com/lihter/Movie-App/pull/28/commits/3a1dd8418b7535a8fac5faf7eea2183f097a6eef#diff-0e250b879344a24765e175866cb828b87d12e258a723eb41dcf4344c1dcd19fdR31-R36)
Did that also for `DetailTitleView` and `RecommendationsView`.